test/integration/tests/crud-entity/common-crud.test.js

```diff
@@ -108,26 +114,45 @@ function testFunc() {
               });
           });
 
-          it('should return all the entities', (done) => {
-            const onNextSpy = sinon.spy();
-            storeToTest.find()
-              .subscribe(onNextSpy, done, () => {
-                try {
-                  utilities.validateReadResult(dataStoreType, onNextSpy, [entity1, entity2], [entity1, entity2, entity3], true);
-                  return utilities.retrieveEntity(collectionName, Kinvey.DataStoreType.Sync, entity3)
-                    .then((result) => {
-                      if (result) {
-                        result = utilities.deleteEntityMetadata(result);
-                      }
-                      expect(result).to.deep.equal(dataStoreType === Kinvey.DataStoreType.Cache ? entity3 : undefined);
-                      done();
-                    })
-                    .catch(done);
-                } catch (error) {
-                  done(error);
-                }
-                return Promise.resolve();
+          it.only('should return all the entities', (done) => {
```

Comment:
It doesn't look like this is testing anything. It looks like it should be testing if all entities are returned.

Reply:
I have created this PR only for testing purposes - to test something in Travis and it is not for a review. I think, that I did not assign it for a review to anyone...

Reply:
Closing the PR - I will modify the travis.yml in my branch in order to test my changes in Travis.